test/controller.test.ts

```diff
@@ -261,31 +253,19 @@ contract('Controller', ([owner, accountOwner1, accountOperator1, random]) => {
         true,
       )
 
-      const batch = (await controller.getBatchDetails(expiredOtoken.address))[0]
-      // set batch oracle
-      await oracle.setBatchOracle(batch, batchOracle.address)
-      // set locking and dispute period
-      await oracle.setLockingPeriod(batchOracle.address, lockingPeriod)
-      await oracle.setDisputePeriod(batchOracle.address, disputePeriod)
-    })
-
-    it('should return false when price is not pushed to Oracle yet', async () => {
-      assert.equal(
-        await controller.isPriceFinalized(expiredOtoken.address),
-        false,
-        'Price is not finalized because it is not stored yet',
-      )
```

Comment:
can you still keep this test on should be false when price has not been finalized?

Reply:
There were originally 2 tests for this:
1. before pushing the price to oracle, this should return false
2. after pushing the price to oracle but during dispute period, this should return false.

But if you look at the `setIsFinalized()` code, it gets the answer from  `oracle.getExpiryPrice(underlying, expiry)` which returns true or false. So we were basically testing:
1.  before pushing the price to oracle, `oracle.getExpiryPrice` should return false
2. after pushing the price to oracle but during dispute period, `oracle.getExpiryPrice` should return false.

Which are not unit tests for controller at all. 
Instead, we just need one test for this: When `oracle.getExpiryPrice` returns false, `isPriceFinalized` should return false. 

Now I simplify the test because I changed the MockOracle contract to make it simpler to mock returned data. (don't need to push price to Oracle, wait for dispute period... etc, just mock the answer of`getExpiryPrice` )